Source/Library/UIColor+Hex.swift

```diff
@@ -0,0 +1,29 @@
+import UIKit
+
+public extension UIColor {
+    
+    convenience init(hex string: String) {
```

Comment:
yes it is a copy of extension from Hue, Also the idea of duplication and fixing twice crossed my mind too but i am kind of leaning towards keeping minimum dependencies. Personal preferences? May be. One function duplication does not justify a full dependency and even more what if bug in Hue breaks Lightbox.

Just trying to explore the thought process of iOS team.